amulet/helpers.py

```diff
@@ -67,7 +67,7 @@ def juju(args, env=None, include_model=True):
         else:
             args = list(args)
             # insert the model arg after the command, but before any other args
-            args[1:0] = [model_flag, default_environment()]
+            args += [model_flag, default_environment()]
```

Comment:
I know @thedac is on holiday until later this week, so I would suggest that we either:

a. Revert the commit which broke Amulet, then have the author re-propose a more complete solution.

or

b. Discard @thedac's PR and have someone propose a new/separate fix to current master.

In either case, we have some urgency as we have a charm development gate down over this.  Please advise with preference.